docs/UserGuide.md

```diff
@@ -423,7 +434,7 @@ Format: `remark INDEX r/[REMARKS]`
 * Adds a remark `REMARKS` to the client at `INDEX`
 * The index refers to the index number shown in the displayed client list.
 * The index **must be a positive integer** 1, 2, 3, …​
-* If there are multiple remark inputs, only the last one will be added.
+* If there are multiple remark inputs `r/Hello r/Test`, only the last one (`r/Test`) will be added.
```

Comment:
Maybe this could be rephrased into something easier to follow for a new reader.

Something like:

* If there are multiple remark inputs (e.g.`r/Remark 1 r/Remark 2`), only the last remark (`r/Remark 2`) will be added.

or

* If there are multiple remark inputs, only the last remark will be added. See example `3` below for a better illustration.

Personally, I prefer the second option, as the formatting of other commands' bullet points do not include examples in them. The second option conforms to the formatting.